packages/dependency-royalty-management/contracts/MultiRoyaltyDistributor.sol

```diff
@@ -68,18 +67,28 @@ abstract contract MultiRoyaltyDistributor is IEIP2981, IMultiRoyaltyDistributor,
     /// @notice EIP 2981 royalty info function to return the royalty receiver and royalty amount
     /// @param tokenId of the token for which the royalty is needed to be distributed
     /// @param value the amount on which the royalty is calculated
-    /// @return address the royalty receiver
-    /// @return value the EIP2981 royalty
-    function royaltyInfo(uint256 tokenId, uint256 value) public view override returns (address, uint256) {
+    /// @return receiver address the royalty receiver
+    /// @return royaltyAmount value the EIP2981 royalty
+    function royaltyInfo(uint256 tokenId, uint256 value)
+        public
+        view
+        override
+        returns (address receiver, uint256 royaltyAmount)
+    {
         (address payable _defaultRoyaltyReceiver, uint16 _defaultRoyaltyBPS) =
             IRoyaltyManager(royaltyManager).getRoyaltyInfo();
         if (_tokenRoyaltiesSplitter[tokenId] != address(0)) {
-            return (_tokenRoyaltiesSplitter[tokenId], (value * _defaultRoyaltyBPS) / TOTAL_BASIS_POINTS);
+            (receiver, royaltyAmount) = (
+                _tokenRoyaltiesSplitter[tokenId],
+                (value * _defaultRoyaltyBPS) / TOTAL_BASIS_POINTS
+            );
+            return (receiver, royaltyAmount);
         }
         if (_defaultRoyaltyReceiver != address(0) && _defaultRoyaltyBPS != 0) {
-            return (_defaultRoyaltyReceiver, (value * _defaultRoyaltyBPS) / TOTAL_BASIS_POINTS);
+            (receiver, royaltyAmount) = (_defaultRoyaltyReceiver, (value * _defaultRoyaltyBPS) / TOTAL_BASIS_POINTS);
+            return (receiver, royaltyAmount);
         }
-        return (address(0), 0);
+        return (receiver, royaltyAmount);
```

Comment:
I think its more confusing to do it that way, aka assigning to variables every time.
Especially on the last line, it was clear that if nothing is found we return `(address(0), 0);`, now it takes some time to figure our.